src/components/Syntax.tsx

```diff
@@ -7,27 +7,31 @@ interface SyntaxProps {
   color: string;
 }
 
-const Syntax: React.FC<SyntaxProps> = ({ title, description, examples, color }) => {
+const Syntax: React.FC<SyntaxProps> = ({
+  title,
+  description,
+  examples,
+  color,
+}) => {
   const navigate = useNavigate();
 
-  const handleExampleClick = (example: string) => {
-    navigate(`/search?query=${example}`);
-  };
-
   return (
-    <div className="p-8 flex flex-col sm:flex-row border-t border-black">
-      <div className="sm:w-1/2 sm:p-4 sm:pr-20 text-center sm:text-left">
-        <h2 className="text-xl font-bold mb-2">{title}</h2>
+    <div className="flex flex-col border-t border-border p-8 sm:flex-row">
```

Comment:
I'm assuming a lot of these changes are just your linter re-ordering the Tailwind right, @KojinKuro?